custom_components/tahoma/__init__.py

```diff
@@ -197,3 +203,8 @@ async def update_listener(hass: HomeAssistant, entry: ConfigEntry):
         coordinator.original_update_interval = new_update_interval
 
         await coordinator.async_refresh()
+
+
+async def is_stateless(devices: List[Device]) -> bool:
+    """Return true if all the devices are stateless."""
+    return all(device.states is None or len(device.states) == 0 for device in devices)
```

Comment:
We could check here for devices with assumed_state as well? What is the benefit of checking for states again?

Reply:
We haven’t yet build the tahoma devices ;)